homeassistant/components/wittiot/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Device IP address",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        },
+        "description": "Please enter the IP address of the device to view the data"
```

Comment:
Please checkout `data_description` fields of other integrations which have a host.